### DISCUSSION TITLE
v1.4.5

### DISCUSSION THREAD

**Discussion by sgkoishi** [sgkoishi/PackTracker#4] [General]
Add option to show hidden untracked packs

<hr /><em>This discussion was created from the release <a href='https://github.com/sgkoishi/PackTracker/releases/tag/v1.4.5'>v1.4.5</a>.</em>

**Comment by sgkoishi** [comment]
Some pack types that were never tracked will be hidden in PityTimers and Statistic automatically in the previous version. It's not bad but currently conflicting with the new feature of v1.4.4 since it hides packs even when the player wants to view the grant counter.

This version added a switch to control the hiding feature and is enabled (not hiding) by default.

**Comment by TheMrKochan** [comment]
congratulations on your first hundred downloads! I wish you to multiply them at least 100 times :)
